cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -163,6 +164,7 @@ webhooks:
       path: /v1/admitlabel
   name: check-ignore-label.gatekeeper.sh
   timeoutSeconds: HELMSUBST_VALIDATING_WEBHOOK_TIMEOUT
+  failurePolicy: HELMSUBST_VALIDATING_WEBHOOK_FAILURE_POLICY
```

Comment:
these 2 should be separate values, they have different defaults

Reply:
I expected that, do we have a recommendation for different names so user can differentiate between both?